python/niizarr/_header.py

```diff
@@ -109,6 +114,25 @@ def update(self, key_values: dict):
             self[key] = value
 
 
+# TODO: JNifTI has the following convention but it is not supported by numpy
+# DTYPES = Recoder([
```

Comment:
Are you sure this formalism isn't compatible with numpy?
I would assume that `np.dtype("uint8")` or `np.dtype([("r", "uint8"), ("g", "uint8"), ("b", "uint8"), ("a", "uint8")]` would work. If it does, let's use the JNifti type specification -- we're only losing the endianness information.